### DISCUSSION TITLE
MySQL Authentication

### DISCUSSION THREAD

**Discussion by belltanellc** [loopbackio/loopback-next#8210] [Q&A]
I see that MySQL 8 defaults to SHA2 instead of native and LB does not implement that plugin. Is there any thought to providing that?

**Comment by achrinza** [comment]
It's something we can look into; If the underlying driver package used by `loopback-conector-mysql` supports it, then it should be possible.

**Reply by belltanellc** [reply]
That would sure be great.

https://dev.mysql.com/doc/refman/8.0/en/pluggable-authentication.html

Thx for considering it!
